hopsworks-common/src/main/java/io/hops/hopsworks/common/proxies/client/HttpClient.java

```diff
@@ -116,4 +117,8 @@ public void setAuthorizationHeader(HttpRequest httpRequest) {
   public <T> T execute(HttpHost host, HttpRequest request, ResponseHandler<T> handler) throws IOException {
     return client.execute(host, request, handler);
   }
+  
+  public <T> T execute(HttpUriRequest request, ResponseHandler<T> handler) throws IOException {
```

Comment:
Why do we need this method? Can't we use execute(HttpHost host, HttpRequest request, ResponseHandler<T> handler) ? I think we're doing the same here https://github.com/logicalclocks/hopsworks/blob/master/hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/jupyter/config/JupyterProcessMgr.java#L271